src/utils/fs.js

```diff
@@ -49,8 +49,16 @@ function _symlink(src: string, dest: string, type: string) {
   return promisify(cb => fs.symlink(src, dest, type, cb));
 }
 
-function cmdShim(src: string, dest: string) {
-  return promisify(cb => _cmdShim(src, dest, cb));
+function stripExtension(filePath: string) {
+  return filePath.substr(0, filePath.length - path.extname(filePath).length);
```

Comment:
```js
path.basename(filePath, path.extname(filePath));
```

Reply:
For input `/project/test.cmd` wouldn't that give just `test`, the shim cmd function requires an absolute path (`/project/test`).

I initially had `path.join(path.dirname(filePath), path.basename(filePath, path.extname(filePath))` but thought that for speed it would be better just to substring the extension off the end. Thoughts?

Reply:
I'd rather use built-in functions than hand optimising something so small

Reply:
@jamiebuilds Done 👍